src/mono/wasm/debugger/BrowserDebugProxy/DebugStore.cs

```diff
@@ -529,6 +561,15 @@ public bool IsLexicallyContainedInMethod(MethodInfo containerMethod)
                     (EndLocation.Line == containerMethod.EndLocation.Line && EndLocation.Column < containerMethod.EndLocation.Column));
     }
 
+    internal record ParameterInfo(string name, ConstantTypeCode? typeCode = null, byte[] value = null)
+    {
+        public string Name => name;
+        public ConstantTypeCode? TypeCode => typeCode;
+        public byte[] Value => value;
+
+        public bool HasDefaultValue = value != null;
+    }
```

Comment:
You only need to add `HasDefaultValue` property.
```suggestion
    internal record ParameterInfo(string Name, ConstantTypeCode? TypeCode = null, byte[] Value = null)
    {
        public bool HasDefaultValue => value != null;
    }
```

The rest will get declared as properties automatically, because this is a `record`.